src/main/java/org/mifos/connector/mpesa/auth/AuthRoutes.java

```diff
@@ -97,6 +103,8 @@ public void configure() {
     }
 
     private String createAuthHeader(String key, String secret) {
+        key = key.replace("\n", "");
+        secret = secret.replace("\n", "");
```

Comment:
Same here as well?

Reply:
here the data is only processed and not logged.